xls/dslx/stdlib/std.x

```diff
@@ -396,6 +396,41 @@ fn round_up_to_nearest_test() {
     assert_eq(u32:4, round_up_to_nearest(u32:4, u32:2));
 }
 
+// Returns `x` rounded up to the nearest multiple of `y`, where `y` is a positive power of 2
+fn round_up_to_nearest_pow2_unsigned<N: u32>(x: uN[N], y: uN[N]) -> uN[N] { (x + y - uN[N]:1) & -y }
```

Comment:
Let's try to drop unsigned as it's already given in the type signature.

(I know right now user defined types can't be parameterized on signedness which is a wart, but I think going with the more convenient name probably makes sense here.)

Reply:
I guess there's precedent in the sizeof_signed and sizeof_unsigned, so nevermind, this LGTM.